config/application.rb

```diff
@@ -23,7 +23,13 @@ class Application < Rails::Application
 
     require "whitehall"
     # Initialize configuration defaults for originally generated Rails version.
-    config.load_defaults 6.1
+    config.load_defaults 7.0
```

Comment:
I think to switch to this default we're going to need some transitory code for cookie rotation.

As per: https://github.com/alphagov/content-tagger/commit/a54df45b37efee1f070855927952465e041d7cd6